[API] DAO - Insert into table fails

Inserting into tables created with DAO-API fails.
```
2021-11-29 07:56:07.778 [INFO ] [http-nio-8080-exec-9] o.e.dirigible.api.v3.core.Console - LOG: Migration track table created.
2021-11-29 07:56:07.827 [TRACE] [http-nio-8080-exec-9] app.db.dao.migrations - Inserting MIGRATIONS entity
2021-11-29 07:56:07.877 [WARN ] [http-nio-8080-exec-9] o.e.d.api.v3.db.DatabaseFacade - Implicitly creating a Sequence [MIGRATIONS_ID] due to: [Sequence """MIGRATIONS_ID""" not found; SQL statement:
SELECT NEXTVAL( '"MIGRATIONS_ID"' ) [90036-200]]
2021-11-29 07:56:07.908 [ERROR] [http-nio-8080-exec-9] app.db.dao.migrations - Inserting {} MIGRATIONS failed
2021-11-29 07:56:07.917 [TRACE] [http-nio-8080-exec-9] app.db.dao.migrations - Rolling back changes after failed MIGRATIONS[null] insert. ```

--- a/modules/database/database-sql/src/main/java/org/eclipse/dirigible/database/sql/dialects/h2/H2NextValueSequenceBuilder.java
+++ b/modules/database/database-sql/src/main/java/org/eclipse/dirigible/database/sql/dialects/h2/H2NextValueSequenceBuilder.java
@@ -41,7 +41,7 @@ public class H2NextValueSequenceBuilder extends NextValueSequenceBuilder {
 	 */
 	@Override
 	public String generate() {
-		String sequenceName = (isCaseSensitive()) ? encapsulate(this.getSequence()) : this.getSequence();
+		String sequenceName = this.getSequence();
 		String sql = format(PATTERN_SELECT_NEXT_VAL_SEQUENCE, sequenceName);
 		return sql;
 	}

